src/pages/tamogatas.hu.tsx

```diff
@@ -0,0 +1,116 @@
+      <p>A független és objektív média támogatásával és a médiatudatosság erősítésével.</p>
+      <p>Valaki mondta, hogy a választás nem lehet szabad, ha az emberek nem dönthetnek szabadon. A szabad
+        döntéshez
+        azonban szükség van arra, hogy a média ne befolyásolja az embereket, hanem hitelesen, tényszerűen tájékoztassa.
+        Nyilván nem lehet teljesen
+        semlegesnek lenni - az akkor lenne, ha nem is írnának egy szót sem - de el lehet érni azt a szintet, amelyet
+        független és
+			objektív újságírásnak lehet nevezni.</p>
+      <p>Egy honlapot fejlesztünk, ahol az érdeklődők megtalálhatnak minden fontos információt az egyes
+        médiumokról:
+        tulajdonosok és politikai kötődéseik, szerkesztők, látogatottsági adatok, világnézet és ideológiai elköteleződés,
+        szavahihetőség és függetlenség. Ezen információk segítségével bárki képes lesz megtalálni a világnézetéhez illő,
+        objektív, tényeket közlő hírforrásokat, az általuk szolgáltatott tartalom segítségével pedig képes lesz dönteni,
+			képes lesz felismerni és gyakorolni jogait, kiállni a jó ügyek mellett.</p>
+      <p><Link to="/hu/rolunk">Bővebben a céljainkról...</Link></p><br /><br />
+    </div>
+    <div id="contribute" className="tabcontent">
+      <h2>Támogass minket!</h2>
+      <p>Sokféleképp tudsz segíteni nekünk, legyen a mód bármelyik, nagyon hálásak leszünk:</p>
+      <ul>
+        <li>Ha tudsz programozni, és jártas vagy a webes technológiákban, akkor forkolj minket a <a href="https://github.com/daroczypal/project-d-."
+          target="_blank">githubon</a>.</li>
+        <li>Kedved lenne ahhoz a lelkes csapathoz tartozni, amelyik ezt az oldalt készíti? Tárt karokkal
+				<a href="mailto:democrable@gmail.com"> várunk</a>.</li>
+        <li>Egy honlap beindítása rengeteg költséggel jár. Támogass minket, hogy jobb, színvonalasabb és
+          bővebb szolgáltatásokat nyújthassunk.</li>
+        <li><Link to="/hu/rolunk#contribution">Még több módja a segítségnek...</Link></li>
+      </ul>
+    </div>
+
+    <div id="fundr">
+      <div className="fundr-grid-item" id="paypal-account">
+        <p>Paypal fiókkal:</p>
+        <span><img id="paypal-logo" alt="paypal logo" src="/assets/paypal-logo-png-22.png" /><a href="https://www.paypal.me/DemocrableHU" >Paypal.me</a></span>
+      </div>
+      <div className="fundr-grid-item" id="bank-account">
+        <p>Bankkártyával:</p>
+        <form action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_top">
+          <input type="hidden" name="cmd" value="_s-xclick" />
+          <input type="hidden" name="hosted_button_id" value="5HFCSKRBMZQ2Q" />
+          <input type="image" src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif" border="0" name="submit"
+            alt="PayPal - The safer, easier way to pay online!" />
+          <img alt="" border="0" src="https://www.paypalobjects.com/hu_HU/i/scr/pixel.gif" width="1" height="1" />
```

Comment:
If an HTML element doesn't exists in TypeScript it should be named data-*.
Here instead of "border" use "data-border". Webpack automatically transforms it to normal.
Unfortunately TypeScript and React doesn't know all HTML properties :(
But we might like to do tests at some points (depending on the scope of the project in the future). This definitely includes a strict typecheck so we shouldn't introduce type errors to master

Reply:
Fixed